src/App.js

```diff
@@ -1,48 +1,42 @@
 import React, { Component } from "react";
-import movieData from "./movieData";
 import TopSection from "./components/TopSection";
 import MovieCardContainer from "./components/MovieCardContainer";
 import SingleMovie from "./components/SingleMovie";
+import apiCalls from "./apiCalls"
 import "./App.css";
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: [],
-      movie: []
+      movie: [],
+      error:""
     };
   }
 
   componentDidMount() {
-    this.setState({
-      movies: movieData.movies
-    })
+    Promise.all([apiCalls.getMovies()])
```

Comment:
I'm a little confused by the use of `Promise.all` here. I would think `getMovies` would return a single promise, which would make the use of `Promise.all` unnecessary, but I'll try to look into that.